doc/source/myrocks/zenfs.rst

```diff
@@ -0,0 +1,80 @@
+.. _zenfs:
+
+=========================================
+RocksDB with ZenFS
+=========================================
+
+Implemented in Percona Server for MySQL 8.0.26-16.
+
+`ZenFS <https://zonedstorage.io/projects/zenfs/>`__ is a file system plugin which places files into zones on a raw zoned block device (ZBD) using the RocksDB File System interface. ZenFS depends on ``libzbd`` and requires a Linux kernel implementation which supports MVMe Zoned namespaces. The kernel must be configured with `zone block device support enabled <https://zonedstorage.io/linux/config/#kernel-configuration>`__. 
+
+The following steps successfully run Percona Server with MySQL on the  ``--rocksdb-fs-uri=zenfs://dev:<short_block_device_name>``:
+
+.. note::
+
+    The ``<block_device_name>`` can have a short name desigination which is the ``<short_block_device_name>``. For example, the ``block_device_name`` is ``/dev/nvme1n2`` and the short name is ``nvme1n2``.
+
+#. Identify your ZBD device, ``<block_device_name>``, with ``lsblk``:
+
+    .. sourcecode:: bash
+    
+     lsblk
+
+#. Create an auxiliary directory for the ZenFS metadata locks, for example, ``/var/lib/mysql_aux``. The directory must be write accessible for the `mysql:mysql` UNIX system user account. Each ZBD must have an individual auxiliary directory.
```

Comment:
What does "auxiliary" mean? Where should this directory be located? Any guidance on which device type is suitable here? Do I need a dedicated SSD for this?

Reply:
Internally, ZenFS file system needs not only the zoned block device (ZBD) itself but and additional directory on a regular (POSIX) filesystem which it will be using internally to resolve file locks/shared access. There are no strict requirements on where such directory should be located. The only requirement is that it should be write-accessible by the same process that runs ZenFS (`mysqld` in our case). Basically, if you run `zenfs mkfs --zbd=... --aux_path=...` under the same Unix user your `mysqld` process will be run, the utility will create the `aux_path` directory with proper file permissions automatically.
In case you have several ZBDs installed on your system, each of them must have a dedicated `aux_path` directory.
These `aux_path` directories are not expected to occupy significant amount of disk space as no file data is stored there - only file structure.

Reply:
Updated.